purchase_open_qty/tests/test_purchase_open_qty.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
+from openerp.fields import Datetime
+
+
+class TestPurchaseOpenQty(TransactionCase):
+    def setUp(self):
+        super(TestPurchaseOpenQty, self).setUp()
+        self.purchase_order_model = self.env['purchase.order']
+        purchase_order_line_model = self.env['purchase.order.line']
+        partner_model = self.env['res.partner']
+        prod_model = self.env['product.product']
+        analytic_account_model = self.env['account.analytic.account']
+        self.product_uom_model = self.env['product.uom']
+
+        pa_dict = {
+            'name': 'Partner 1',
+            'supplier': True,
+        }
+        self.partner = partner_model.sudo().create(pa_dict)
+        po_dict = {
+            'partner_id': self.partner.id,
+        }
+        # PURCHASE ORDER NUM 1
```

Comment:
Why are you screaming?  :fearful: I don't like sentences in high caps, but it's up to you.